extensions/ecc/sw-macros/src/lib.rs

```diff
@@ -98,6 +99,7 @@ pub fn sw_declare(input: TokenStream) -> TokenStream {
                 fn #sw_double_extern_func(rd: usize, rs1: usize);
                 fn #hint_decompress_extern_func(rs1: usize, rs2: usize);
                 fn #hint_non_qr_extern_func();
+                fn #setup_sw_extern_func();
```

Comment:
noting this does setup for both add_ne and double_ne (this is the way it was originally), which makes sense because you would basically never use just one over the other